users/views.py

```diff
@@ -0,0 +1,103 @@
+import re, bcrypt, jwt, json, string, secrets, random
+from jwt                  import algorithms
+
+from django.http.response import JsonResponse
+from django.views         import View
+
+from users.models         import Member
+from my_settings          import SECRET_KEY
+
+class SignupView(View):
+    def post(self, request):
+        try:
+            data      = json.loads(request.body)
+            Id        = re.compile("[a-z0-9]{5,19}$")
+            Password  = re.compile("[0-9]{8,16}")
+            Email     = re.compile("^[a-zA-Z0-9+-_.]+@[a-zA-z0-9-]+\.[a-zA-z0-9-]+$")
+            
+            if  Password.match(data['password'])  is None or Id.match(data['account']) is None or Email.match(data['email']) is None:
+                return JsonResponse({'message':' INVALID_FORMAT'}, status=400)
+            
+            if Member.objects.filter(account = data["account"]).exists():
+                return JsonResponse({"message": "Already_Exist"}, status=400)
+
+            encoded_pw = bcrypt.hashpw(data['password'].encode('utf-8'), bcrypt.gensalt())
+
+            point = random.randrange(10000, 1000000)
+
+            Member.objects.create(
+                account      = data['account'],
+                password     = encoded_pw.decode('utf-8'),
+                name         = data['name'],
+                address      = data['address'],
+                phone_number = data['phone_number'],
+                email        = data['email'],
+                points       = point
+             )
```

Comment:
```suggestion
            
            Member.objects.create(
                account      = data['account'],
                password     = encoded_pw.decode('utf-8'),
                name         = data['name'],
                address      = data['address'],
                phone_number = data['phone_number'],
                email        = data['email'],
                points       = random.randrange(10000, 1000000)

             )
```
이 정도는 바로 작성하셔도 좋을 거 같습니다.

Reply:
수정사항 반영해서 Commit 했습니다!